contrib/pg_tde/documentation/docs/setup.md

```diff
@@ -124,43 +124,25 @@ Load the `pg_tde` at the start time. The extension requires additional shared me
 
 ## WAL encryption configuration (tech preview)
 
-After you [enabled `pg_tde`](#enable-extension) and started the Percona Server for PostgreSQL, a principal key and a keyring for WAL are created. Now you need to instruct `pg_tde ` to encrypt WAL files by configuring WAL encryption. 
+Before turning WAL encryption on, you must first create a principal key for WAL.
 
-Here's how to do it:
+Here's what to do:
 
-1. Enable WAL level encryption using the `ALTER SYSTEM` command. You need the privileges of the superuser to run this command:
+1. Create pg_tde extesion if it is not exists:
```

Comment:
```suggestion
1. Create the `pg_tde` extension if it doesn't exist yet:
```

Reply:
Do we need to mention where to create the extension? It'll be created for the current db, right? Will it affect WAL encryption anyhow?

Reply:
We need to create extension to use functions from the next steps (pg_tde_add_global_key_provider_vault_v2 etc). If we assume that extension should've been created on some previous steps than we can commit this step. But it is safer have it I think